apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -668,12 +668,15 @@ export default function teacherSections(state = initialState, action) {
       }
     }
 
-    // Selecting Course 1-4 or A-F should auto-enable Stage Extras.
+    // Selecting the following courses should auto-enable Stage Extras:
+    //  Course 1-4, A-F, Express, and Pre-reader Express
     if (action.props.scriptId) {
       const script =
         state.validAssignments[assignmentId(null, action.props.scriptId)];
       stageExtraSettings.stageExtras = !!(
-        script && /course[1-4a-f]/.test(script.script_name)
+        script &&
+        (/course[1-4a-f]/.test(script.script_name) ||
+          /express/.test(script.script_name))
```

Comment:
This approach makes me sad and will be difficult to maintain. this info should really live as a property in the script. then when we create copies for new version years, this property will be copied over, i.e. if express-2018 has the flag, and you create express-2019 from it, then express-2019 would already have the flag. 

Could we please take this opportunity to clean this up? The `stage_extras_available` property is already defined on many scripts, so all we would need to do is add it to the .script files for all the courses which currently match these regexes, and then remove this test.

Reply:
oh, perfect! i hadn't known about the `stage_extras_available` property -- that's a much better way. i'll update and re-request review